src/engraving/dom/anchors.cpp

```diff
@@ -111,10 +111,14 @@ TimeTickAnchor* EditTimeTickAnchors::createTimeTickAnchor(Measure* measure, Frac
             continue;
         }
 
-        Staff* linkedStaff = toStaff(linkedObj);
-        Measure* linkedMeasure = linkedStaff->score()->tick2measureMM(measure->tick());
+        Staff* linkedStaff = linkedObj == staff ? staff : toStaff(linkedObj);
```

Comment:
This line is redundant (`toStaff(linkedObj)` will also be `== staff` if `linkedObj == staff`).